W-22513567 sf RNs: May 20 (#3558)
forcedotcom/cli: releasenotes/README.md

* [Salesforce CLI Plugin Developer Guide](https://github.com/salesforcecli/cli/wiki/Quick-Introduction-to-Developing-sf-Plugins)
* [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)

> ## 2.136.6 (May 27, 2026) [stable-rc]

Reviewer: ```suggestion
## 2.136.8 (May 27, 2026) [stable-rc]
```

These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.

------------

> * NEW and CHANGED: To improve security, we updated Salesforce CLI to reduce the accidental exposure of sensitive credentials (access tokens, SFDX Auth URLs, and user passwords.) Here's a summary of the changes, but see **\<ADD LINK TO KA OR PINNED ISSUE HERE>** for additional details, why we made these changes, and the timeline,: 

Reviewer: ```suggestion
* NEW and CHANGED: To improve security, we updated Salesforce CLI to reduce the accidental exposure of sensitive credentials (access tokens, SFDX Auth URLs, and user passwords.) Here's a summary of the changes, but see [forcedotcom/cli/issues/3560](https://github.com/forcedotcom/cli/issues/3560) for additional details, why we made these changes, and the timeline,: 
```

    * We removed sensitive credentials from the output of these CLI commands:
        - `org display`
        - `org list --json `

>         - `org auth list --json`

Reviewer: ```suggestion
        - `org create scratch --json`
        - `org resume scratch --json`        
```

>         - `org display user `

Reviewer: ```suggestion
        - `org display user [--json]`
```

        - `org list users --json`

>     * To avoid breaking your existing CI/CD pipelines, we added a temporary environment variable (`SF_ORG_DISPLAY_SHOW_SECRETS`) that overrides the changes to these existing commands.

Reviewer: ```suggestion
    * To avoid breaking your existing CI/CD pipelines, we added a temporary environment variable that overrides the changes to these existing commands. To use, set `SF_TEMP_SHOW_SECRETS=true`
```

* [Salesforce CLI Plugin Developer Guide](https://github.com/salesforcecli/cli/wiki/Quick-Introduction-to-Developing-sf-Plugins)
* [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)

## 2.136.8 (May 27, 2026) [stable-rc]

These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.

------------

* NEW and CHANGED: To improve security, we updated Salesforce CLI to reduce the accidental exposure of sensitive credentials (access tokens, SFDX Auth URLs, and user passwords.) Here's a summary of the changes, but see [forcedotcom/cli/issues/3560](https://github.com/forcedotcom/cli/issues/3560) for additional details, why we made these changes, and the timeline,: 

    * We removed sensitive credentials from the output of these CLI commands:
        - `org display`
        - `org list --json `
        - `org create scratch --json`
        - `org resume scratch --json`        
        - `org display user [--json]`

>         - `org list users --json`

Reviewer: ```suggestion
        - `org list users --json`
        - `org login jwt --json`
        - `org login web --json`
        - `org login sfdx-url --json`
        - `org login access-token --json`
        - `org list auth --json`
```

* [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)

## 2.136.8 (May 27, 2026) [stable-rc]

These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.

------------

* NEW and CHANGED: To improve security, we updated Salesforce CLI to reduce the accidental exposure of sensitive credentials (access tokens, SFDX Auth URLs, and user passwords.) Here's a summary of the changes, but see [forcedotcom/cli/issues/3560](https://github.com/forcedotcom/cli/issues/3560) for additional details, why we made these changes, and the timeline,: 

    * We removed sensitive credentials from the output of these CLI commands:
        - `org display`
        - `org list --json `
        - `org create scratch --json`
        - `org resume scratch --json`        
        - `org display user [--json]`
        - `org list users --json`
        - `org login jwt --json`
        - `org login web --json`
        - `org login sfdx-url --json`
        - `org login access-token --json`
        - `org list auth --json`

>     * To avoid breaking your existing CI/CD pipelines, we added a temporary environment variable that overrides the changes to these existing commands. To use, set `SF_TEMP_SHOW_SECRETS=true`

Reviewer: ```suggestion
    * To avoid breaking your existing CI/CD pipelines, we added a temporary environment variable (`SF_TEMP_SHOW_SECRETS`) that overrides the changes to these existing commands.
```
